packages/mux-player-react/src/lazy.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import Suspense from './BrowserOnlySuspense';
+import { MuxPlayerProps, MuxPlayerRefAttributes } from './index';
+
+const MuxPlayerIndex = React.lazy(() => import('./index'));
+
+const Fallback = (props: Omit<MuxPlayerProps, 'playerSoftwareVersion' | 'playerSoftwareName'>) => {
+  const { style, ...rest } = props;
+  return (
+    <>
+      {/* TODO: can we add mux-player to JSX.IntrinsicElements */}
+      {/* @ts-ignore */}
+      <mux-player
+        data-mux-player-react-placeholder
+        style={{
```

Comment:
Let's say a user is trying to style the component with styled-components, via className. Would placing styles here override those styles? Are we preventing the user from styling their mux-player? If so, we'll need to figure out how to define these styles in a lower-specificity way :(